pkg/cmd/run.go

```diff
@@ -396,6 +386,19 @@ func defaultFallbackFile(token string, project string, config string) string {
 	return filepath.Join(defaultFallbackDir, fileName)
 }
 
+// generate the passphrase used for encrypting a secrets file
+func getPassphrase(cmd *cobra.Command, flag string, config models.ScopedOptions) string {
+	if cmd.Flags().Changed(flag) {
+		return cmd.Flag(flag).Value.String()
+	}
+
+	if config.EnclaveProject.Value != "" && config.EnclaveConfig.Value != "" {
+		return fmt.Sprintf("%s:%s:%s", config.Token.Value, config.EnclaveProject.Value, config.EnclaveConfig.Value)
+	}
+
+	return config.Token.Value
```

Comment:
Not sure if this matters, but should this be `fmt.Sprintf("%s", localConfig.Token.Value)`?

Reply:
Those two statements are equivalent, so I went with the simpler one